test/discr/operator_utilities_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str, super
+
+# External module imports
+import pytest
+import numpy as np
+
+# ODL imports
+import odl
+from odl.discr.operator_utilities import matrix_representation
+from odl.util.testutils import almost_equal
+
+
+class MultiplyOp(odl.Operator):
```

Comment:
It's called `MatVecOperator` and [located in `ntuples.py`](https://github.com/odlgroup/odl/blob/master/odl/space/ntuples.py#L1289)